geopyspark/geotrellis/layer.py

```diff
@@ -435,14 +435,18 @@ def to_geotiff_rdd(self,
 
         return create_python_rdd(result, ser)
 
-    def to_spatial_layer(self, target_time=None):
+    def to_spatial_layer(self, target_time=None, merge_duplicates=False):
```

Comment:
I want to push back on this parameter here, it does not do anything different from subsequent `. merge_duplicate_keys` call and is a lot less clear because its a boolean.

It would result in lines like
```py
layer.to_spatial_layer(target_time, True)
```
vs 
```py
layer.to_spatial_layer(target_time). merge_duplicate_keys()
```